src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/JsonElementTests.cs

```diff
@@ -102,48 +41,6 @@ public void SerializeJsonElementArray()
             obj.Verify();
         }
 
-        public class JsonElementArrayClass : ITestClass
```

Comment:
Have these been moved elsewhere?

Reply:
Yup they were moved to `tests/Common/TestClasses/TestClasses.cs` for easier sharing with the src-gen test project.